applications/fine_tuning/template.yaml

```diff
@@ -0,0 +1,118 @@
+version: v1
+volumes:
+  scratch:
+    ingress:
+      - destination:
+          uri: file://{{ .DataPrepScript }}
+        source:
+          uri: {{ .GitHubRepoUrl }}/{{ .DataPrepScript }}
+      - destination:
+          uri: file://{{ .ModelLoadScript }}
+        source:
+          uri: {{ .GitHubRepoUrl }}/{{ .ModelLoadScript }}
+      - destination:
+          uri: file://{{ .FineTuningScript }}
+        source:
+          uri: {{ .GitHubRepoUrl }}/{{ .FineTuningScript }}
+      - destination:
+          uri: file://{{ .TestingScript }}
+        source:
+          uri: {{ .GitHubRepoUrl }}/{{ .TestingScript }}
+    reference: volume://user/ephemeral/scratch
```

Comment:
Do you think for consistency and reusability the volume reference should be a parameter?